src/pages/index.astro

```diff
@@ -0,0 +1,722 @@
+                even: isEven(index)
+              })}
+              id={categoryName}
+            >
+              <header class="comparisons-header">
+                <h2 class="category-name">
+                  <a href={`#${categoryName}`}>{titleCase(categoryName)}</a>
+                </h2>
+                {category.alternatives && (
+                  <div class="alternatives">
+                    <h4 class="alternatives-title">Alternatives:</h4>
+                    <ul class="alternatives-list">
+                      {Object.entries(category.alternatives).map(
+                        ([alternativeName, alternativeUrl]) => (
+                          <li>
+                            <a
+                              href={alternativeUrl}
+                              target="_blank"
+                              rel="noopener"
+                              class="alternative-link"
+                            >
+                              {alternativeName}
+                            </a>
+                          </li>
+                        )
+                      )}
+                    </ul>
+                  </div>
+                )}
+              </header>
+              <div class="category-comparisons">
+                {Object.entries(category.comparisons).map(
+                  ([comparisonName, comparison]) => (
+                    <div class="comparison" id={comparisonName}>
+                      <header class="comparisons-header">
+                        <h3 class="comparison-title">
+                          <a href={`#${comparisonName}`}>
+                            {titleCase(comparisonName.replaceAll('_', ' '))}
+                          </a>
+                        </h3>
+                        {comparison.alternatives && (
+                          <div class="alternatives">
+                            <h4 class="alternatives-title">Alternatives:</h4>
+                            <ul class="alternatives-list">
+                              {Object.entries(comparison.alternatives).map(
+                                ([alternativeName, alternativeUrl]) => (
+                                  <li>
+                                    <a
+                                      href={alternativeUrl}
+                                      target="_blank"
+                                      rel="noopener"
+                                      class="alternative-link"
+                                    >
+                                      {alternativeName}
+                                    </a>
+                                  </li>
+                                )
+                              )}
+                            </ul>
+                          </div>
+                        )}
+                      </header>
+                      <div class="engines">
+                        {Object.entries(comparison.engines).map(
```

Comment:
Since this is a centered flexbox, the item layouts look a little strange when multiple comparisons render. For example, if there are only two comparisons, it looks like this:
![image](https://user-images.githubusercontent.com/10366495/188780812-5512c950-c489-4357-92a0-2dbd904504df.png)
Whereas three comparisons assume a different layout:
![image](https://user-images.githubusercontent.com/10366495/188780869-97d0180b-641a-42e3-a79c-23d78f82ad1c.png)

Reply:
Rather than rendering all of the engines together equally, can we split out the jQuery engine to render on the left and each "comparison" to render on the right. Also would it make sense here to reintroduce the version toggle IFF the "Do you need to support IE" checkbox is true?